libraries/operation-graph/src/WatchLoop.ts

```diff
@@ -193,7 +193,9 @@ export class WatchLoop implements IWatchLoopState {
 
             try {
               status = await this.runUntilStableAsync(abortController.signal);
-              this._requestRunPromise.finally(requestRunFromHost);
+              this._requestRunPromise.finally(() => {
+                requestRunFromHost('runIPCAsync');
```

Comment:
@dmichon-msft what is the appropriate name to supply here?

(If we're going to be referring to requesters/operations by names, then we should provide a meaningful name in every code path, rather than printing awkward messages like "This job was started by an unknown source.")

Reply:
If `.finally` is not forwarding the promise result (which is the underlying requester), use `.then` instead. The error code path is unreachable code, since the `reject` callback is discarded.